webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const Path = require('path');
+const webpackConfig = require('@silverstripe/webpack-config');
+const {
+  moduleCSS,
+  pluginCSS,
+} = webpackConfig;
+
+const ENV = process.env.NODE_ENV;
+const PATHS = {
+  MODULES: 'node_modules',
+  FILES_PATH: '../',
+  ROOT: Path.resolve(),
+  SRC: Path.resolve('client/src'),
+  DIST: Path.resolve('client/css'),
```

Comment:
Done by building the file twice, since I can't use the webpack copy plugin to copy files _after_ they're built and a `cp` in the package.json build script wouldn't be used by CI.